spec/mailers/previews/devise_invitable/mailer_preview.rb

```diff
@@ -2,7 +2,7 @@
 module DeviseInvitable
   class MailerPreview < ActionMailer::Preview
     def invitation_instructions
-      Devise::Mailer.invitation_instructions(User.last, "faketoken")
+      Devise.mailer.invitation_instructions(User.last, "faketoken")
```

Comment:
Fixed the Preview page. http://localhost:3000/rails/mailers/devise_invitable/mailer/invitation_instructions